sip-core/src/main/java/de/ikor/sip/foundation/core/trace/CustomTracer.java

```diff
@@ -21,19 +23,26 @@
    * @param traceHistory {@link TraceHistory}
    * @param exchangeFormatter {@link SIPExchangeFormatter}
    * @param camelContext {@link CamelContext}
+   * @param traceType int
    */
   public CustomTracer(
       TraceHistory traceHistory,
       SIPExchangeFormatter exchangeFormatter,
-      CamelContext camelContext) {
+      CamelContext camelContext,
+      SIPTraceTypeEnum traceType) {
```

Comment:
SIPTraceTypeEnum is not a bean, so it will cause an error. You could use SIPTraceConfig instead.